### PR TITLE
Add a very basic compare classification to unblock fulfillment testing

### PR DESCRIPTION
**Pull request by shifucun**
Note this is only used in /nlnext but not /nl, so existing scripts should not be affected.

### CODE REVIEW

**Review by pradh — APPROVED** [review]
Thanks for the addition!

File: server/lib/nl/nl_detection.py [hunk 1]
```diff
@@ -178,8 +184,11 @@ class ClassificationType(IntEnum):
 
 # The supported classifications in order. Later entry is preferred.
 RANKED_CLASSIFICATION_TYPES = [
-    ClassificationType.SIMPLE, ClassificationType.CONTAINED_IN,
-    ClassificationType.RANKING, ClassificationType.CORRELATION
+    ClassificationType.SIMPLE,
+    ClassificationType.COMPARE,
```
[the remark this answers is not in the document]
Comment (juliawu) [reply server/lib/nl/nl_detection.py:188]: +1

[the remark this answers is not in the document]
Comment (shifucun) [reply server/lib/nl/nl_detection.py]: Done

**Review by juliawu — APPROVED** [review]
Thanks for adding!

**Pull request merged**